esi_leap/tests/api/controllers/v1/test_node.py

```diff
@@ -64,3 +64,57 @@ def test_get_all(self, mock_gpl, mock_lga, mock_oga, mock_gnl):
         self.assertEqual(data['nodes'][0]['lessee'], 'fake-project')
         self.assertEqual(data['nodes'][0]['properties'], {
             'cpu': '40'})
+
+    @mock.patch('esi_leap.common.ironic.get_node_list')
+    @mock.patch('esi_leap.common.keystone.get_project_list')
+    def test_get_resource_class_filter(self, mock_gpl, mock_gnl):
```

Comment:
Can you rename this to `test_get_all_resource_class_filter`?

Reply:
Sure I will do this.